Library/Formula/couchdb.rb

```diff
@@ -32,7 +38,7 @@ def install
     # in the welcome message
     inreplace "etc/couchdb/default.ini.tpl.in" do |s|
       s.gsub! "%package_author_name%", "Homebrew"
-      s.gsub! "%version%", "%version%-1"
+      s.gsub! "%version%", "%version%-3"
```

Comment:
Can you use `pkg_version` here instead which gets you the version plus revision?